docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,56 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
 </runtime>
 ```
 
+## What's new in accessibility in .NET Framework 4.8.1
+.NET Framework 4.8.1 includes new accessibility features in the following areas:
+
+- [Windows Forms](#winforms481)
+
+- [Windows Presentation Foundation (WPF)](#wpf481)
+
+<a name="winforms481"></a>
+
+### Windows Forms
+
+**Added and improved UIA representations**
+
+Prior to .NET Framework 4.8.1, Windows Forms was missing support for a range of UIA patterns to support assistive technology, for example, to interact with applications. This deficiency could cause Narrator or other screen readers to report incomplete or incorrect information. It could also affect important functionality like moving a cursor through the text in a <xref:System.Windows.Forms.TextBox> control. With .NET Framework 4.8.1, all of the required patterns for the common controls have been implemented. This new functionality gives users of assistive technology a much richer application interaction experience.
```

Comment:
```suggestion
Prior to .NET Framework 4.8.1, Windows Forms was missing support for a range of UIA patterns to support assistive technology to interact with applications. This deficiency could cause Narrator or other screen readers to report incomplete or incorrect information. It could also affect important functionality like moving a cursor through the text in a <xref:System.Windows.Forms.TextBox> control. With .NET Framework 4.8.1, all of the required patterns for the common controls have been implemented. This new functionality gives users of assistive technology a much richer application interaction experience.
```

Remove `for example`